cmd/bosun/sched/sched.go

```diff
@@ -588,11 +598,38 @@ func (s *Schedule) action(user, message string, t models.ActionType, st *models.
 			return "", fmt.Errorf("cannot acknowledge closed alert")
 		}
 		st.NeedAck = false
+	case models.ActionCancelClose:
+		found := false
+		for i, a := range st.Actions {
```

Comment:
Decided to make it so an additional delayed close on an alert while a there is a pending delayed close updates the close time. This is because you might want to close multiple alerts with multi-select to close them, and having one error the group would bad for the user workflow.

Main downside is that this doesn't get reflected in the log of actions as a new entry, but I'm okay with missing this instance vs adding yet another action possibility or action field.